Documentation/install/upgrade.rst

```diff
@@ -332,6 +332,11 @@ Annotations:
 
   - ``global.hubble.tls.enabled=false``
   - ``global.hubble.tls.auto.enabled=false``
+* Cilium has upgraded its CRDs to v1, from v1beta1. Users who are upgrading
+  from Kubernetes below v1.16.0 to past Kubernetes v1.16.0, while running
```

Comment:
>Users who are upgrading
  from Kubernetes below v1.16.0 to past Kubernetes v1.16.0, while running
  Cilium v1.9 or greater, should run the pre-flight checker mentioned above.
  The pre-flight check will ensure the custom resources installed inside the
  cluster are well-formed.

this whole section can be removed. When performing the upgrade, users should run the pre-flight build regardless of the information is written here.

Reply:
That's a good point. We don't really need the extra contraints. We should detect badly formed CRDs regardless of K8s version.